packages/aila/src/core/Aila.test.ts

```diff
@@ -115,36 +118,48 @@ describe("Aila", () => {
           useAnalytics: false,
           useModeration: false,
         },
-        plugins: [],
         services: {
-          chatCategoriser: mockCategoriser as unknown as AilaCategorisation,
+          chatLlmService: new MockLLMService(),
         },
+        plugins: [],
       });
 
       await ailaInstance.initialise();
 
-      expect(mockCategoriser.categorise).toHaveBeenCalledTimes(1);
+      ailaInstance.document.content = {
+        ...ailaInstance.document.content,
+        title: "Roman Britain",
+        subject: "history",
+        keyStage: "key-stage-2",
+        topic: "The Roman Empire",
+      };
```

Comment:
It would be useful to have a comment explaining this, as it seems to be overwriting the result of the test